CMakeLists.txt

```diff
@@ -1043,9 +1043,8 @@ function(TRANSFORM_MAKEFILE_INC INPUT_FILE OUTPUT_FILE)
   string(REPLACE "$(top_srcdir)"   "\${CURL_SOURCE_DIR}" MAKEFILE_INC_TEXT ${MAKEFILE_INC_TEXT})
   string(REPLACE "$(top_builddir)" "\${CURL_BINARY_DIR}" MAKEFILE_INC_TEXT ${MAKEFILE_INC_TEXT})
 
-  string(REGEX REPLACE "\\\\\n" "�!�" MAKEFILE_INC_TEXT ${MAKEFILE_INC_TEXT})
+  string(REGEX REPLACE "\\\\\n" "" MAKEFILE_INC_TEXT ${MAKEFILE_INC_TEXT})
```

Comment:
The replacement string should be " " instead of "" to be safe in cases where you have

```
XY_FILES = progress.h\
formdata.h
```

which would get converted to

```
XY_FILES = progress.hformdata.h
```

Even if this is not happening in current Makefile.inc files, an author of such a file expects that a whitespace is set when writing `\↵`.